### PR TITLE
Milestone 2- Basic Components

### PR DESCRIPTION
**Pull request by george-shammar**
This Pull Request seeks to merge the following features to the default branch:

1. App Component
2. Button Component
3. ButtonPanel Component
4. Display Component

These Components have been implemented with basic features as per the requirements of Milestone-2

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Button from './Button';
+
+class ButtonPanel extends React.Component {
+/* eslint-disable */
```
Comment (addod19) [inline src/components/ButtonPanel.js:5]: Kindly avoid inline disabling rules, it is almost the same as altering the config files

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  renderButton(i) {
+    return (
+      <Button name={i} />
+    );
+  }
+  /* eslint-enable */
```
Comment (addod19) [inline src/components/ButtonPanel.js:11]: Kindly avoid inline disabling rules, it is almost the same as altering the config files

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @george-shammar, congratulations on this milestone :fireworks: The PR and README is professional :smile: Commit messages easy to understand :smile: The App, Button, ButtonPanel, and the Display components implemented as expected :+1: The right linter config files were used :1st_place_medal: Awesome
However, there is an inline ESLint disabling rule in the ButtonPanel component which needs to be removed. Kindly check the comment below

Once this is resolved, kindly submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 
  

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**